# The SelfDetection service returns the master controllerâ€˜ state and the ok flag.

#request constants
---
#response constants
bool ok
int8 states 
